package.json

```
@@ -7,7 +7,9 @@
     "dev": "webpack serve --config webpack.config.js",
     "build": "webpack --config webpack.config.js",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```

kalau command build nya diganti aja gimana ? biar seragam sama yang lain, dipisahin pake `:`

```suggestion
    "storybook:build": "build-storybook"
```

---

okee, kayaknya emang lebih baik dipisah pake `:`